### PR TITLE
Update packages: greetings-german and greetings-english

### PR DESCRIPTION
**Pull request by katrinleinweber**
Since the hub-site can has a `Source` tab, I see no point in keeping the tables and associated dev dependencies.

Also:

- migrating 2 snippets as per https://github.com/katrinleinweber/espanso-greetings-german/issues/1#issuecomment-2450286609
- deduplicating all `clipboard` uses into 1 `global_vars`

### CODE REVIEW

**Comment by smeech** [comment]
Minor update to a couple of short, simple trigger/replace packages. No script/shell/cmd or malicious code.

I must admit, I liked the `README.md` tables - I consider that page in the Hub as a sort of shop-window for the package with a quick and easy way of seeing what it's about, without inspecting the code. I guess it's less important for easily readable packages like yours. I'm not sure that people browsing the packages will stop to run the `curl` command, and most of us don't have `yq` installed anyway. I wasn't sure of the relevance of the bit about dependencies, though, so am happy to lose that.

Would you kindly update the package paths and [`_manifest.yml`](https://espanso.org/docs/packages/package-specification/#the-_manifestyml-file) versions to `/0.2.0/` please?

By the way, you might be interested in the short conversation on [Discord](https://discord.com/channels/884163483409731584/1013914627886817372/1288900938630955088) on a similar topic

@federico-terzi isn't around much these days, so I wouldn't expect a response from him, I'm afraid.

Thank you for your submission!

File: packages/greetings-english/0.1.0/_manifest.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 author: Katrin Leinweber
 description: A few greeting snippets
-homepage: https://github.com/katrinleinweber/espanso-greetings-english
+homepage: https://github.com/espanso/hub/tree/main/packages/greetings-english/0.1.0
```
Comment (smeech) [inline packages/greetings-english/0.1.0/_manifest.yml:3]: Would you kindly update the package paths and [`_manifest.yml`](https://espanso.org/docs/packages/package-specification/#the-_manifestyml-file) versions to `0.1.1` or `0.2.0` please, as this is an update?

File: packages/greetings-german/0.1.0/_manifest.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 author: Katrin Leinweber
 description: A few greeting snippets
-homepage: https://github.com/katrinleinweber/espanso-greetings-german
+homepage: https://github.com/espanso/hub/tree/main/packages/greetings-german/0.1.0
 name: greetings-german
 title: Greetings in German
 version: 0.1.0
```
Comment (smeech) [inline packages/greetings-german/0.1.0/_manifest.yml:6]: Would you kindly update the package paths and [`_manifest.yml`](https://espanso.org/docs/packages/package-specification/#the-_manifestyml-file) versions to `0.1.1` or `0.2.0` please, as this is an update?

**Review by smeech — CHANGES_REQUESTED** [review]

**Comment by katrinleinweber** [comment]
@smeech Sure thing, done.

> I liked the `README.md` tables - I consider that page in the Hub as a sort of shop-window for the package …

Agreed. How about a TypeScript-based enhancement of [the `Source` tab](https://github.com/espanso/hub-frontend/blob/477b44641c0fb5716ea3dbbb24721ce55a71fcd1/pages/%5BpackageName%5D/v/%5Bversion%5D.tsx#L365-L373) so that it would not only display the raw YAML, but actually be rendered below the ReadMe content into a table?

**Comment by smeech** [comment]
> Agreed. How about a TypeScript-based enhancement of [the `Source` tab](https://github.com/espanso/hub-frontend/blob/477b44641c0fb5716ea3dbbb24721ce55a71fcd1/pages/%5BpackageName%5D/v/%5Bversion%5D.tsx#L365-L373) so that it would not only display the raw YAML, but actually be rendered below the ReadMe content into a table?

Is this a suggested enhancement for the Hub, or a plan to reorganise your own `README.md`? 
If the latter, we have to ensure the Hub registers its content. We have enough difficulty rendering `README.md` files, so I guess they have to be fairly conventional.

**Review by smeech — APPROVED** [review]

**Comment by smeech** [comment]
Merged. I'll be interested to see how the Hub copes with deletion of the previous version (which I think is generally a good idea).

**Pull request merged**